sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -1739,6 +1836,12 @@ pi_result piextUSMSharedAlloc(void **ResultPtr, pi_context Context,
     return PI_INVALID_OPERATION;
   }
 
+  // 'Size' must be power of two in order to prevent memory corruption
+  // error
+  if ((Size & (Size - 1)) != 0) {
```

Comment:
sycl/include/sycl/ext/intel/experimental/esimd/detail/util.hpp
Contains isPowerOf2()

Reply:
`Size` argument here is runtime argument while corresponding argument for `getNextPowerOf2` is template argument. Can I use the latter for this PI_API with runtime argument?

Reply:
Hi. What I mean is it's always good to keep common utilities common. Yes, there's no runtime variant for nextPowerOf2 in sycl utils currently, but it can always be added there?

Reply:
Done. Thanks for suggestion.